### PR TITLE
Add Option to select Firewall SKU

### PR DESCRIPTION
**Pull request by marvinbuss**
## Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

1. Adds Option to select Firewall SKU

### Breaking Changes

1. None

## Testing Evidence

image

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: eslzArm/eslz-portal.json [hunk 1]
```diff
@@ -1463,6 +1463,32 @@
                                 ]
                             }
                         },
+                        {
+                            "name": "esAzFwSku",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Select Azure Firewall Sku",
```
Comment (daltondhcp) [inline eslzArm/eslz-portal.json:1469]: Can we change the label from 'Sku' to 'tier' so we are using the same taxonomy in the regular firewall deployment experience?
image

**Comment by daltondhcp** [comment]
Apart from above comment - tested deployments for both vWAN and H&S and works well! 👏

File: eslzArm/eslz-portal.json (hunk 1 above)
Comment (marvinbuss) [reply eslzArm/eslz-portal.json]: DONE

**Review by daltondhcp — APPROVED** [review]
LGTM ⚡

**Pull request merged**